### PR TITLE
feat: added cube icon

### PR DESCRIPTION
**Pull request by OKendigelian**
We use the Cube icon for Product Manager links in the Vertical Booking Pages widget, which is not currently available in the Picasso library.
The goal of this ticket is to add this icon to the library.
The SVG for this icon is below.

```
<svg
  viewBox="0 0 24 24"
  width="24"
  height="24"
  xmlns="http://www.w3.org/2000/svg"
  xmlns:xlink="http://www.w3.org/1999/xlink"
>
  <path d="M10.5002 0.996582L21.0049 5.99917V17.9956L10.5002 23.0022L0 17.9956V5.99917L10.5002 0.996582ZM1 7.02858V17.3649L10.002 21.6566V11.3156L1 7.02858ZM20.004 7.02858L11.002 11.3146V21.6546L20.004 17.3639V7.02858ZM10.5 2.10388L1.744 6.27558L10.501 10.4456L19.259 6.27558L10.5 2.10388Z" />
</svg>
```

Figma link https://www.figma.com/file/TqaGgbpjGSUDf7qq153Isq/Product-Iconography?node-id=622%3A2

[FX-2806]

### Description

Describe the changes and motivations for the pull request.

### How to test

- FIXME: Add the steps describing how to verify your changes

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| Insert screenshots or screen recordings | Insert screenshots or screen recordings |

### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- [x] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [x] Annotate all `props` in component with documentation
- [x] Create `examples` for component
- [x] Ensure that deployed demo has expected results and good examples
- [x] Ensure that tests pass by running `yarn test`
- [x] Ensure that visuals tests pass by running `yarn test:visual`. If not - check the documentation [how to fix visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md#fixing-broken-visual-tests-inside-a-pr)
- [x] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [x] Self reviewed
- [x] Covered with tests

**Breaking change**

- [x] codemod is created and showcased in the changeset
- [x] test alpha package of Picasso in StaffPortal

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-2806]: https://toptal-core.atlassian.net/browse/FX-2806?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by TomasSlama** [comment]
@OKendigelian Hello 👋, thanks for your contribution, we need also link to the base design (Figma) with this Icon, so we can check the correctness

**Comment by OKendigelian** [comment]
> @OKendigelian Hello 👋, thanks for your contribution, we need also link to the base design (Figma) with this Icon, so we can check the correctness

Hi there! Sure https://www.figma.com/file/TqaGgbpjGSUDf7qq153Isq/Product-Iconography?node-id=622%3A2

File: packages/picasso/src/Icon/index.ts [hunk 1]
```diff
@@ -93,6 +93,8 @@ export { default as Copy16 } from './Copy16'
 export { default as Copy24 } from './Copy24'
 export { default as CreditCard16 } from './CreditCard16'
 export { default as CreditCard32 } from './CreditCard32'
+export { default as Cube16 } from './Cube16'
```
Comment (TomasSlama) [inline packages/picasso/src/Icon/index.ts:96]: Designers named it `Box`, I think we should be consistent and use the same name.

File: .changeset/large-ways-arrive.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@toptal/picasso': minor
+---
+
+Added Cube icon
```
Comment (TomasSlama) [inline .changeset/large-ways-arrive.md:5]:
```suggestion
Add Cube icon
```
We should write changesets the same as commits. In present simple.

**Review by TomasSlama — COMMENTED** [review]
I have left just few small comments, otherwise it looks good to me 👍

File: packages/picasso/src/Icon/index.ts (hunk 1 above)
Comment (OKendigelian) [reply packages/picasso/src/Icon/index.ts]: Done!

File: .changeset/large-ways-arrive.md (hunk 2 above)
Comment (OKendigelian) [reply .changeset/large-ways-arrive.md]: Done!

File: .changeset/large-ways-arrive.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+'@toptal/picasso': minor
+---
+
+Add Cube icon
```
Comment (TomasSlama) [inline .changeset/large-ways-arrive.md:5]:
```suggestion
Add Box icon
```

**Review by TomasSlama — APPROVED** [review]

**Review by denieler — APPROVED** [review]

**Comment by ozgurkececioglu** [comment]
Code looks 👌 but I'm curious about the SVG image, as you can also realize that it is not totally aligned horizontally. Is it intended?

Screen Shot 2022-05-25 at 11 54 22

**Review by separatio — APPROVED** [review]

**Review by TomasSlama — CHANGES_REQUESTED** [review]
@ozgurkececioglu is right.

In design 
image
there is horizontal padding 1px from left, 2px from right.

But in our case it is not horizontally centered
image
0px from left, 3px from right.

**Review by TomasSlama — APPROVED** [review]
Base team [approved](https://toptal-core.slack.com/archives/CCC3GP6CC/p1653482547512899) the little bit smaller size of the Icon

**Review by ozgurkececioglu — APPROVED** [review]

**Review by augustobmoura — APPROVED** [review]

**Pull request merged**